R/variant.R

```diff
@@ -38,12 +39,11 @@ Variant <- R6::R6Class(
     #' @param to Targeting.
     send_mail = function(to = c("me", "collaborators", "collaborators_viewers")) {
       warn_experimental("send_mail")
-      if (length(to) > 1) to <- "me"
-      url <- glue::glue("variants/{self$get_variant()$id}/sender")
+      url <- unversioned_url("variants", self$get_variant()$id, "sender")
       self$get_connect()$POST(
         path = url,
         body = list(
-          email = to
+          email = match.arg(to)
```

Comment:
If we wanted to not allow partial matches (especially since the second is a substring of the last) we already have `rlang` elsewhere, we could use [`arg_match()`](https://rlang.r-lib.org/reference/arg_match.html)

Reply:
`match.arg()` is probably fine here (and certainly better than how this was handled before) but yeah `arg_match()` is probably better.